src/virtualenv/seed/embed/base_embed.py

```diff
@@ -28,9 +28,7 @@ def __init__(self, options) -> None:
 
         if options.no_wheel:
             warn(
-                "The --no-wheel option is deprecated. "
-                "It has no effect, wheel is no longer bundled in virtualenv. "
-                "This option will be removed in pip 26.",
+                "The --no-wheel option is deprecated. It has no effect, wheel is no longer bundled in virtualenv.",
```

Comment:
When the user is running with 3.8 it should still work though no?

Reply:
It has no effect with 3.8 too. We removed all the code to ever bundle `wheel`.

Reply:
We removed `wheel` from `src/virtualenv/seed/wheels/embed/`, so it won't be bundled.

And setuptools doesn't need it, but `pip --no-use-pep517` won't work, because it requires `wheel` to enable legacy mode.

Reply:
So, does this mean we either:

 - add wheel back to 3.8
 - remove support for 3.8 envs

?

Reply:
yeah we need to add it back because we didn't drop the support for 3.8 it just

Reply:
OK, let me do that.

Reply:
Done. It brings a bit of unavoidable mess, but trivial to revert later.